app/controllers/page_controller.rb

```diff
@@ -3,6 +3,16 @@ def show
     page_slug = params[:page_slug] ? params[:page_slug] : 'home'
     @page = Page.includes(:page_group).find_by(slug: page_slug.downcase, page_groups: {slug: params[:page_group_friendly_id].downcase})
     @page_components = @page.page_components
+    @page_components.each do |pc|
+      if pc.component_type == "PageMapComponent"
+        @practices_list = PageMapComponent.find_by_id(pc.component_id)
+        @short_name = @practices_list.short_name
+        adoptions = helpers.get_adopting_facilities_for_these_practices(@practices_list, @practices_list.display_successful_adoptions, @practices_list.display_in_progress_adoptions, @practices_list.display_unsuccessful_adoptions)
+        build_map_component adoptions
+        @adoptions_count = adoptions.count
```

Comment:
Do we need this? We aren't using it anywhere.

Reply:
Yeah originally we were using it.. so I probably left it in in case we changed our mind.. lol